src/services/to-dos.service.cpp

```diff
@@ -0,0 +1,47 @@
+#include "to-dos.service.h"
+
+Json::Value ToDo::toJson() const {
```

Comment:
I preferred for the service to return the `ToDo` structure, and for the controller to map it to Json in every method